### PR TITLE
Keep a change log

### PR DESCRIPTION
**Pull request by kriskowal**
This change converts the change log to a standard format.

### CODE REVIEW

**Comment by kriskowal** [comment]
Ah, no, coming right up ;)

File: CHANGELOG.md [hunk 1]
```diff
@@ -5,104 +5,97 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
...
   now reserved. Override the names of these fields in the generated code with
   the `go.name` annotation.
 - Plugins: The version of ThriftRW used to compile the plugin is now matched
   against the version actually generating code.
 
-
 ## [0.5.0] - 2016-11-10
-
-- **Breaking**: Generated enums now have first class JSON support. Enums are
+### Changed
```
Comment (prashantv) [inline CHANGELOG.md]: I think the `go.name` annotation was might be an `Added`?

Comment (prashantv) [reply CHANGELOG.md]: similarly, `Plugins:` has a couple that have "Added" and "Removed".

**Comment by abhinav** [comment]
Thanks @kriskowal. I made the changes requested by Prashant and moved some of the entries around as well.

**Pull request merged**